config/src/main/resources/application.docker.conf

```diff
@@ -87,7 +96,7 @@ ubirchAvatarService {
   }
 
   akka {
-    actorTimeout = 10
+    actorTimeout = ${AKKA_ACTOR_TIMEOUT} // seconds
```

Comment:
did you adjust the helm charts accordingly already? But good idea to externalise that config.

Reply:
I will adjust later with the configuration of trackle serivce.